### PR TITLE
Change the proxy format in CC.

### PR DESCRIPTION
**Pull request by lubronzhan**
### Which issue(s) this PR fixes

Fixes #https://github.com/vmware-tanzu/tanzu-framework/issues/3813

### Describe testing done for PR

Refactor the proxy settings in CC 

### Release note

```release-note
Change the proxy setting in CC template
```

### CODE REVIEW

**Comment by alfredthenarwhal** [comment]
Tests failed! Build no: 3085

File: providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml [hunk 1]
```diff
@@ -142,24 +142,27 @@ spec:
                 type: string
               node:
                 type: string
-          proxy:
```
Comment (tenczar) [inline providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml:145]: I would bump the version number on the annotation at line 12 of this file `identifier.cc.vmware.com` and update Shimon on the UX team that the format of the variables has changed. We need to do the same for each clusterclass.

File: providers/infrastructure-vsphere/v1.4.1/yttcc/overlay.yaml [hunk 2]
```diff
@@ -131,7 +131,7 @@ spec:
     variables:
     #@ vars = get_vsphere_vars()
     #@ for configVariable in vars:
-    #@  if vars[configVariable] != None and configVariable in ["apiServerPort", "controlPlane", "worker", "vcenter", "VSPHERE_WINDOWS_TEMPLATE", "apiServerEndpoint", "network", "imageRepository", "trust", "user", "auditLogging", "aviAPIServerHAProvider", "ntpServers", "TKR_DATA"]:
+    #@  if vars[configVariable] != None and configVariable in ["apiServerPort", "controlPlane", "worker", "vcenter", "VSPHERE_WINDOWS_TEMPLATE", "apiServerEndpoint", "network", "imageRepository", "trust", "user", "auditLogging", "aviAPIServerHAProvider", "ntpServers", "TKR_DATA", "proxy"]:
```
Comment (tenczar) [inline providers/infrastructure-vsphere/v1.4.1/yttcc/overlay.yaml:134]: Doesn't this change need to go into Azure and AWS as well?

**Review by tenczar — CHANGES_REQUESTED** [review]
just a couple small changes to make and make sure that the UX team knows about the change to the variable structure

File: providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml (hunk 1 above)
Comment (lubronzhan) [reply providers/infrastructure-aws/v2.0.0-beta.1/cconly/base.yaml:145]: Got it. tahnks

File: providers/infrastructure-vsphere/v1.4.1/yttcc/overlay.yaml (hunk 2 above)
Comment (lubronzhan) [reply providers/infrastructure-vsphere/v1.4.1/yttcc/overlay.yaml:134]: Yeah you are right, let me update those

**Comment by alfredthenarwhal** [comment]
Tests failed! Build no: 3095

**Comment by alfredthenarwhal** [comment]
Build failed! Build no: 3100

**Review by tenczar — APPROVED** [review]
LGTM

**Comment by alfredthenarwhal** [comment]
Tests failed! Build no: 3106

**Comment by alfredthenarwhal** [comment]
Tests failed! Build no: 3121

**Pull request merged**